### DISCUSSION TITLE
[Feature] Allow loop/repeat of videos

### DISCUSSION THREAD

**Discussion by natedawg** [immich-app/immich#3289] [Feature Request]
### The feature

Since immich is meant to manage videos taken from a mobile device a lot of the videos are most likley short clips. It would be nice to have a loop/repeat for clips. This can be done on the web with right click however the mobile app has no option for it. It would be nice to allow it to be enabled by default on both platforms.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by etnoy** [comment]
Agreed, I think looping by default is reasonable for immich

**Reply by bo0tzz** [reply]
Looping by default would be a trivial change [here](https://github.com/immich-app/immich/blob/main/web/src/lib/components/asset-viewer/video-viewer.svelte#L26-L32). Having a button to switch is unfortunately less straightforward, as the default html5 video player doesn't include that.
